service/core/kafka/client/src/main/resources/play/reference-overrides.conf

```diff
@@ -1,3 +1,4 @@
 akka.kafka.consumer.kafka-clients {
   auto.offset.reset = "earliest"
+  max.in.flight.requests.per.connection = 1
```

Comment:
It might make sense to repeat the comment from the Kafka docs:
> The maximum number of unacknowledged requests the client will send on a single connection before blocking. Note that if this setting is set to be greater than 1 and there are failed sends, there is a risk of message re-ordering due to retries (i.e., if retries are enabled).

Possibly with a link to http://kafka.apache.org/documentation/#producerconfigs

Reply:
Right, this is actually a **Producer** setting. So it needs to go into `akka.kafka.producer.kafka-clients`.